confuse.py

```diff
@@ -936,7 +945,9 @@ def _add_user_source(self):
         """
         filename = self.user_config_path()
         if os.path.isfile(filename):
-            self.add(ConfigSource(load_yaml(filename) or {}, filename))
+            self.add(ConfigSource(load_yaml(filename,
+                                            loader=self.Loader) or {},
+                                  filename))
```

Comment:
Perhaps easier-to-read formatting:
```suggestion
            self.add(ConfigSource(
                load_yaml(filename, loader=self.Loader) or {},
                filename,
            ))
```